translations/harbour-advanced-camera-fi.ts

```diff
@@ -0,0 +1,300 @@
+    <message>
+        <source>Automatic Scene Detection</source>
+        <translation>Automaattinen kohtauksen tunnistus</translation>
+    </message>
+    <message>
+        <source>Augmented Reality</source>
+        <translation>Lisätty todellisuus</translation>
+    </message>
+    <message>
+        <source>Backlight</source>
+        <translation>Vastavalo</translation>
+    </message>
+    <message>
+        <source>Barcode</source>
+        <translation>Viivakoodi</translation>
+    </message>
+    <message>
+        <source>Beach</source>
+        <translation>Ranta</translation>
+    </message>
+    <message>
+        <source>Candlelight</source>
+        <translation>Kynttilänvalo</translation>
+    </message>
+    <message>
+        <source>Fireworks</source>
+        <translation>Ilotulitus</translation>
+    </message>
+    <message>
+        <source>Flowers</source>
+        <translation>Kukat</translation>
+    </message>
+    <message>
+        <source>HDR</source>
+        <translation>HDR</translation>
+    </message>
+    <message>
+        <source>Landscape</source>
+        <translation>Maisema</translation>
+    </message>
+    <message>
+        <source>Large Aperture</source>
+        <translation>Suuri aukko</translation>
+    </message>
+    <message>
+        <source>Night</source>
+        <translation>Yö</translation>
+    </message>
+    <message>
+        <source>Night Portrait</source>
+        <translation>Yömuotokuva</translation>
+    </message>
+    <message>
+        <source>Party</source>
+        <translation>Juhlat</translation>
+    </message>
+    <message>
+        <source>Portrait</source>
+        <translation>Muotokuva</translation>
+    </message>
+    <message>
+        <source>Small Aperture</source>
+        <translation>Pieni aukko</translation>
+    </message>
+    <message>
+        <source>Snow</source>
+        <translation>Lumi</translation>
+    </message>
+    <message>
+        <source>Sports</source>
+        <translation>Urheilu</translation>
+    </message>
+    <message>
+        <source>Spotlight</source>
+        <translation>Kohdevalo</translation>
+    </message>
+    <message>
+        <source>Steady Photo</source>
+        <translation>Vakaa kuva</translation>
+    </message>
+    <message>
+        <source>Sunset</source>
+        <translation>Auringonlasku</translation>
+    </message>
+    <message>
+        <source>Theatre</source>
+        <translation>Teatteri</translation>
+    </message>
+    <message>
+        <source>Action</source>
+        <translation>Toiminta</translation>
+    </message>
+    <message>
+        <source>Off</source>
+        <translation>Pois</translation>
+    </message>
+</context>
+<context>
+    <name>FlashModel</name>
+    <message>
+        <source>Unknown</source>
+        <translation>Tuntematon</translation>
+    </message>
+    <message>
+        <source>Auto</source>
+        <translation>Auto</translation>
```

Comment:
Should these "Auto" modes all be "Automaattinen"?

Reply:
I tried to think what would be better regarding the UI look and feel. It can be changed to "Automaattinen" if it suits better.

Reply:
It's fine for me this way also.